ExampleAzureFunctionapp/ManualTriggerFunctionCS/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    
+    {
+      "name": "$return",
+      "type": "sendGrid",
```

Comment:
Should this be `SendGrid`, based on the Output Binding description in the README.md?